_overviews/jdk-compatibility/overview.md

```diff
@@ -20,14 +20,30 @@ Sometimes new JVM and JDK (Java Development Kit) versions require us to update S
 | 11 (LTS)    | 3.0.0, 2.13.0, 2.12.4, 2.11.12                               |
 | 8 (LTS)     | 3.0.0, 2.13.0, 2.12.0, 2.11.0                                |
 
-**Using latest patch version of Scala is always recommended**
+Using latest patch version of Scala is always recommended.
 
-Even when a version combination isn't listed as supported, most features may still work.
-
-In general, Scala works on JDK 11+, including GraalVM, but may not take special advantage of features that were added after JDK 8. See [below](#jdk-11-compatibility-notes).
+Even when a version combination isn't listed as supported, most features might still work.
 
 Lightbend offers [commercial support](https://www.lightbend.com/lightbend-platform-subscription) for Scala 2. The linked page includes contact information for inquiring about supported and recommended versions.
 
+## Build tool compatibility table
+
+| JDK version | sbt             |
```

Comment:
```suggestion
| JDK version | Minimum SBT version  |
```

Or however many spaces are needed here

Reply:
s/SBT/sbt/.

Also I'd recommend the similar blurb as Scala and say "Using the latest stable version of sbt is always recommended."